spec/models/course_spec.rb

```diff
@@ -68,6 +68,19 @@
     its(:has_vacancies?) { should be false }
   end
 
+  context "with sites" do
```

Comment:
Sorry if this comes off as picky, but to continue the crusade to get our specs to actually _sound_ right 😛 ... we should have a describe here, for example `describe '#sites'` or `describe 'sites association'`. And I'm not sure the `context` is adding much here, since we only have one ... if you really want it then it should be within the describe block I think.